hub/api/tests/test_chunk_sizes.py

```diff
@@ -99,3 +104,13 @@ def test_extend_and_append(memory_ds):
     _assert_num_chunks(images, 20)
 
     assert len(ds) == 400
+
+
+def test_clear(memory_ds):
```

Comment:
Add a test in a test_api.py as well, which tests values of len(ds), len(ds.tensor) etc.